craft_archives/repo/apt_sources_manager.py

```diff
@@ -76,9 +83,11 @@ class AptSourcesManager:
     def __init__(
         self,
         *,
-        sources_list_d: Path = Path("/etc/apt/sources.list.d"),  # noqa: B008
+        sources_list_d: Path = _DEFAULT_SOURCES_DIRECTORY,
```

Comment:
If you make it a constant, the function call happens up where you're declaring the constant rather than when declaring the function. 

Both flake8-bugbear and ruff have a set of immutable calls (`tuple`, `frozenset`, `re.compile`, etc.) and a [way to extend that list](https://beta.ruff.rs/docs/settings/#flake8-bugbear) if we need it. Arguably since paths are immutable it makes sense to include them in the list, but I think the intention is to encourage people to move things like this into constants.

Reply:
ah that makes sense. It also makes sense to me to have `Path` be part of that set, maybe we'll do that piecemeal in the repos